help/dependencies.txt

```diff
@@ -100,12 +166,12 @@ If you want to upgrade Lucene to a newer build proceed like the following:
   queued)
 - remember the build number of Jenkins (left side, first build in list,
   prefixed by '#')
-- Edit ./versions.props and change Lucene's version to '9.0.0-prereleaseX',
+- Edit gradle/libs.versions.toml and change Lucene's version to '9.0.0-prereleaseX',
```

Comment:
When I look at versions.props we don't seem to be doing  this. Is this a legacy from pre-split? Maybe this section should go away since Lucene definitely marches ahead on it's own now? (Main is not using 10.0 let alone a pre-release)